packages/react-magma-dom/src/components/Stepper/Step.tsx

```diff
@@ -0,0 +1,242 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import { CheckIcon, CrossIcon } from 'react-magma-icons';
+
+import { ThemeContext } from '../../theme/ThemeContext';
+import { ThemeInterface } from '../../theme/magma';
+import { useIsInverse } from '../../inverse';
+import { transparentize } from 'polished';
+import { BreakPointStyle } from './Stepper';
+import { HiddenLabelText } from '../Checkbox';
+
+/**
+ * @children required
+ */
+export interface StepProps extends React.HTMLAttributes<HTMLDivElement> {
+  breakpointStyle?: BreakPointStyle;
+  /*
+   * Index set by Stepper component
+   */
+  currentStep?: number;
+  /**
+   * Error state for each step
+   * @default false
+   */
+  hasError?: boolean;
+  /**
+   * Hides label and secondaryLabel
+   * @default false
+   */
+  hideLabels?: boolean;
+  /**
+   * Hides label and secondaryLabel in place of summary view
+   * @default false
+   */
+  summaryView?: boolean;
+  /**
+   * Label beneath each step
+   */
+  label?: string;
+  /**
+   * Sub label beneath each step
+   */
+  secondaryLabel?: string;
+  /**
+   * @internal
+   */
+  stepStatus?: StepStatus;
+  isInverse?: boolean;
```

Comment:
isInverse isn't used at the Step level, so we should remove it.

Reply:
It's used in the styling of `Step` throughout the component, I can make it `@internal`.